packages/peregrine/lib/talons/CreateAccount/useCreateSignIn.js

```diff
@@ -0,0 +1,12 @@
+import { useMutation } from '@apollo/client';
+
+/**
+ * GraphQL mutations for the sign user after create
+ * @property {GraphQLAST} signInMutation mutation for signing
+ */
+export const useCreateSignIn = props => {
+    const { signInMutation } = props;
+    return useMutation(signInMutation, {
+        fetchPolicy: 'no-cache'
+    });
```

Comment:
I think you'd want to move the `fetchPolicy` or other options for that matter to the implementation side of `useCreateSignIn` this gives more control to the developer.